Services/Store/Store.API/Controllers/StoreSectionController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Store.Application.StoreSections.Commands.CreateStoreSection;
+using Store.Application.StoreSections.Commands.DeleteStoreSection;
+using Store.Application.StoreSections.Commands.UpdateStoreSection;
+
+namespace Store.API.Controllers
+{
+    [ApiController]
+    [Route("api/[controller]")]
+    public class StoreSectionController : ApiControllerBase
+    {
+        [HttpPost]
+        public async Task<ActionResult<Guid>> Create(CreateStoreSectionCommand command)
+        {
+            return await Mediator.Send(command);
+        }
+
+        [HttpPut]
+        public async Task<ActionResult> Update(UpdateStoreSectionCommand command)
+        {
+            await Mediator.Send(command);
+
+            return NoContent();
```

Comment:
Sometimes it is better to return updated entity, may be useful for client application performance. Not critical for now though